packages/backend/src/api/resources/JSON.ts

```diff
@@ -205,6 +205,7 @@ export interface SignInTokenJSON extends ClerkResourceJSON {
   user_id: string;
   token: string;
   status: 'pending' | 'accepted' | 'revoked';
+  url: string;
```

Comment:
❓ @nikosdouvlis shouldn't we also add `url` to to `SignInResource` in `packages/types/src/signIn.ts` ?

Reply:
@dimkl This is for the sign in **token** object, not the sign in object. Hope I haven't missed anything.

Reply:
you are correct . sorry.